4-projects/modules/infra_pipelines/main.tf

```diff
@@ -116,6 +122,7 @@ resource "google_cloudbuild_trigger" "non_main_trigger" {
     _BILLING_ID           = var.billing_account
     _DEFAULT_REGION       = var.default_region
     _GAR_REPOSITORY       = local.gar_name
+    _TERRAFORM_IMAGE      = "${local.image_default_region}-docker.pkg.dev/${local.image_project_id}/${local.image_gar_repo_name}/terraform"
```

Comment:
We also seem to override this in cloudbuild-*.yaml? IIUC we can remove that from the yaml since it's being injected here.

Reply:
Indeed. I removed _TERRAFORM_IMAGE from cloudbuild-*.yaml files.